src/Compilers/Core/CodeAnalysisTest/Diagnostics/OperationTests.cs

```diff
@@ -104,5 +106,42 @@ public void IDynamicObjectCreationExpression_PublicExtensionMethodTests()
 
             TestCore(createDynamicExpression);
         }
+
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void TestFlowAnalysisFeatureFlag()
+        {
+            var source = @"
+class C
+{
+    void M()
+    {
+    }
+}";
+            var tree = CSharpSyntaxTree.ParseText(source);
+            TestFlowAnalysisFeatureFlagCore(tree, expectException: true);
+
+            var options = tree.Options.WithFeatures(new[] { new KeyValuePair<string, string>("flow-analysis", "true") });
+            tree = tree.WithRootAndOptions(tree.GetRoot(), options);
+            TestFlowAnalysisFeatureFlagCore(tree, expectException: false);
+        }
+
+        private static void TestFlowAnalysisFeatureFlagCore(SyntaxTree tree, bool expectException)
```

Comment:
Nit: consider making this a local function (rename to camelCase and remove tree parameter), since only used in one test.

Reply:
Done